src/main/java/com/microsoft/info/Constants.java

```diff
@@ -0,0 +1,22 @@
+package com.microsoft.info;
+
+/** Multi-purpose constants holder used accross the SDK */
+public final class Constants {
+    private Constants() {
+    }
+    // Constants for functional tests
+    // TO-DO: document how to register an application for functional
+    // testing purposes
+    /** The application ID to use for unit testing */
+    public static final String APPID = "app-id";
+    /** The user email to use for unit testing */
+    public static final String USERNAME = "user@email.com";
+    /** The user password to use for unit testing */
+    public static final String PASSWORD = "password";
+    /** The tenant ID to use for unit testing */
+    public static final String TENANTID = "tenantid";
+    /** The client secret to use for unit testing */
+    public static final String CLIENTSECRET = "clientsecret";
```

Comment:
```suggestion
```

I don't believe we need those constants anymore since the tests have been removed

Reply:
I thought it would be good to keep them incase we need them for future tests

Reply:
I don't think we should have any unit or integration test at the service library level.
We have unit tests in our conversion lib and generator to cover the accuracy of the generation.
We have integration tests at the raptor level.
But manually testing generated code is counter productive IMHO.